showcase.json

```diff
@@ -0,0 +1 @@
+[{"message":"https://www.youtube.com/watch?v=8rB2TQSPLuA hello","author":"Silverdragon_12"},{"message":"https://www.youtube.com/watch?v=8rB2TQSPLuA hello","author":"Silverdragon_12"},{"message":"https://www.youtube.com/watch?v=8rB2TQSPLuA hello","author":"Silverdragon_12"},{"message":"https://www.youtube.com/watch?v=Waabw_tp0LY hello","author":"Silverdragon_12"}]
```

Comment:
Shouldn't it be empty? I think you forgot :)

Reply:
I did not mean to upload that, sorry ( I will delete soon)